lists/th.csv

```diff
@@ -1,340 +1,373 @@
+http://www.parliament.go.th/,GOVT,Government,2016-09-20,,"Parliament of Thailand. Since 2014 Coup, home of National Reform Steering Assembly"
+http://www.pataninews.net,NEWS,News Media,2016-09-20,,News from Thailand Deep South
+http://www.rtarf.mi.th,GOVT,Government,2016-09-20,,Royal Thai Armed Forces HQ
+http://www.senate.go.th,GOVT,Government,2016-09-20,,Senate of Thailand
+http://www.stou.ac.th,GOVT,Government,2016-09-20,,Open university
+http://www.telecomasia.net,NEWS,News Media,2016-09-20,,"Telecom Asia, also cover ICT news in Thailand"
+http://www.thaigov.go.th,GOVT,Government,2016-09-20,,Royal Thai Government
+http://www.thaivisa.com,NEWS,News Media,2016-09-20,,Expat news
+http://www.thaivisa.com/forum/,GRP,Social Networking,2016-09-20,,Expat activities
+http://www.thaivolunteer.org,ECON,Economics,2016-09-20,,"NGO that supports works of social volunteers, including human rights lawyers project"
+http://www.tnews.co.th,NEWS,News Media,2016-09-20,,News agency
+https://facthai.wordpress.com/,HUMR,Human Rights Issues,2016-09-20,,Anti-censorship group
+https://freedom.ilaw.or.th,HUMR,Human Rights Issues,2016-09-20,,Freedom of Expression Documentation Center
+https://ilaw.or.th,HUMR,Human Rights Issues,2016-09-20,,"People participation in legislation process, rule of law"
+https://storylog.co,HOST,Hosting and Blogging Platforms,2016-09-20,,Thai blogging platform
+https://th.wikipedia.org,NEWS,News Media,2016-09-20,,Thai Wikipedia (HTTPS)
+https://thainetizen.org,HUMR,Human Rights Issues,2016-09-20,,NGO on internet and human rights
+https://tlhr2014.wordpress.com/,HUMR,Human Rights Issues,2016-09-20,,Thai Lawyers for Human Rights
+https://uclthailand.blogspot.com/,HUMR,Human Rights Issues,2016-09-20,,Union of Civil Liberty
+https://www.amnesty.or.th/,HUMR,Human Rights Issues,2016-09-20,,Amnesty International Thailand
+https://www.facebook.com/DemocratPartyTH,POLR,Political Criticism,2016-09-20,,"Democrat Party, a major political party"
+https://www.facebook.com/newdemocracymovement,POLR,Political Criticism,2016-09-20,,Political activists active after the 2014 Coup
+https://www.facebook.com/pheuthaiparty,POLR,Political Criticism,2016-09-20,,"Pheu Thai Party, a major political party"
+https://www.facebook.com/somsakjeam,POLR,Political Criticism,2016-09-20,,Thai history academic in exile
+https://www.facebook.com/uclthailand,HUMR,Human Rights Issues,2016-09-20,,"Union of Civil Liberty, Facebook page"
+https://www.fccthai.com,NEWS,News Media,2016-09-20,,Foreign Correspondents' Club of Thailand 
+https://www.ilaw.or.th,HUMR,Human Rights Issues,2016-09-20,,"People participation in legislation process, rule of law"
+https://www.prachamati.org,NEWS,News Media,2016-09-20,,Civil society group promote the understanding of the Draft Charter Referendum
+https://www.thainetizen.org,HUMR,Human Rights Issues,2016-09-20,,NGO on internet and human rights
```

Comment:
Can the source column be added to these entries?

It doesn't have to be an organisation, but can also just be an alias or just "anonymous".

Also the file should terminate with a newline.